zero-conf-cln.py

```diff
@@ -22,47 +22,6 @@ def on_openchannel(openchannel, plugin, **kwargs):
         return {'result': 'continue'}
 
 
-@plugin.hook('custommsg')
-def on_custommsg(peer_id, payload, plugin, **kwargs):
-    # Decode payload
-    payload_bytes = bytes.fromhex(payload)
-
-    # Check msg type
-    msg_type = int.from_bytes(payload_bytes[:2], byteorder='big', signed=False)
-    if msg_type != 55443:
-        return {'result': 'continue'}
-
-    msg_contents = json.loads(payload_bytes[2:])
```

Comment:
looks like you aren't using json anymore but you're still importing it. 

how exactly does this file get kept in sync with https://github.com/voltagecloud/lsp-server/pull/110/files#diff-528d353e5e12a8eac93e1ab6a376a7006cc215aded669894cc551ab9dd57fd30 ? Looks like it should just be deleted from the lsp-server repo again yeah?

Reply:
fixed, and yeah, it's been deleted there now.